### ISSUE TITLE
After reboot not falling back to node red anymore

### ISSUE THREAD

**Issue by RdeLange** [JarbasAl/fallback-node-red#3]
Hi,
I have succesfully installed the fallback skill on the latest release of piCroft. Everything was running fine after first installation. (tried the 'what does verge say' question succesfully) After reboot of the Picroft though it is not falling back anymore to the node red skill. it tries Wolfram and then goes directly to the UnknownSkill.
Same behavior on my desktop install. It works directly after first installation, but it looks like that it is changing settings somewhere in the first run resulting in not working anymore after a restart (of skill services)
Just to be clear, it is only the fallback part. For instance 'ping node red' is still working, so the skill is loaded.

Any ideas?

Thanks

**Comment by nodered** [comment]
I am having the same problem with the fallback (node-red.ask), stops working after a while.  The converse method does work though.

**Comment by RdeLange** [comment]
Found the solution for this problem.
It has to do with the priority setting. After the first time it will take the value from the skill config file (which is synced/connected/filled from the mycroft,ai backend). The parameter should be given in the registerfunction as an integer. After I changed that piece of code in the skill code into int(self.settings["priority"]) it was working perfectly.
I have not (yet) many expierence with contributing to code via Git (should learn, i know). maybe nodered or jarbas can apply this fix to the code.
